composer.json

```diff
@@ -16,7 +16,7 @@
         "php": "^7.3 || ^8.0",
         "guzzlehttp/guzzle": "^7.0",
         "nesbot/carbon": "^2.0",
-        "illuminate/support": "^5.8 || ^6.0 || ^7.0 || ^8.0"
+        "illuminate/support": "^5.8 || ^6.0 || ^7.0 || ^8.0 | ^9.0"
```

Comment:
```suggestion
        "illuminate/support": "^5.8 || ^6.0 || ^7.0 || ^8.0 || ^9.0"
```

Reply:
Of course it will works but I think suggested change will be better.

Reply:
I agree, thank you!

Reply:
Have you tried to contact with code owner about merging? Yesterday I sent email to Adam. Wait for him reaction.

Reply:
Yes I've done that and gotten no reply, hope you have more luck :)